misc/python/materialize/checks/scenarios.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         # upgrade scenarios can specify another base version.
         self.check_objects = [
             check_class(self.base_version(), self.rng)
-            for check_class in self.checks()
+            for check_class in check_classes
```

Comment:
Can we rename `check_classes` to something that expresses the (or "a") difference to `self.checks()`? I don't have a good suggestion, but maybe just something like `selected_checked_classes` or `filtered_check_classes`?

Reply:
selected_checked_classes it is